src/docker-client.ts

```diff
@@ -21,7 +21,7 @@ type DockerodeExposedPorts = { [port in PortString]: {} };
 
 export interface DockerClient {
   pull(repoTag: RepoTag): Promise<void>;
-  create(repoTag: RepoTag, env: Env, boundPorts: BoundPorts): Promise<Container>;
+  create(repoTag: RepoTag, env: Env, boundPorts: BoundPorts, cmd?: Command[]): Promise<Container>;
```

Comment:
As this will always be an empty array, could we make it a required field?

Reply:
Sure thing, I can remove it. 
The reason why I proposed this was to introduce no breaking changes to the API.